kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/ConsumerGroupService.java

```diff
@@ -157,6 +157,19 @@ private Mono<List<ConsumerGroupDescription>> loadSortedDescriptions(ReactiveAdmi
             .map(descriptions ->
                 sortAndPaginate(descriptions.values(), comparator, pageNum, perPage, sortOrderDto).toList());
       }
+      case MESSAGES_BEHIND -> {
+        Comparator<InternalConsumerGroup> comparator = Comparator.comparingLong(icg ->
+            icg.getMessagesBehind() == null ? 0L : icg.getMessagesBehind());
+        var groupNames = groups.stream().map(ConsumerGroupListing::groupId).toList();
+        yield ac.describeConsumerGroups(groupNames)
+              .flatMap(descriptionsMap ->
+                  getConsumerGroups(ac, descriptionsMap.values().stream().toList())
+                    .map(internalGroups ->
+                      sortAndPaginate(internalGroups, comparator, pageNum, perPage, sortOrderDto).toList())
+                    .map(internalGroups ->
+                      internalGroups.stream().map(InternalConsumerGroup::getDescription).toList())
+              );
+      }
```

Comment:
no need to add description into InternalConsumerGroup
here how it can be done
```
      case MESSAGES_BEHIND -> {
        record GroupWithDescr(InternalConsumerGroup g, ConsumerGroupDescription d) { }

        Comparator<GroupWithDescr> comparator = Comparator.comparingLong(gwd ->
            gwd.g.getMessagesBehind() == null ? 0L : gwd.g.getMessagesBehind());

        var groupNames = groups.stream().map(ConsumerGroupListing::groupId).toList();

        yield ac.describeConsumerGroups(groupNames)
            .flatMap(descriptionsMap -> {
                  List<ConsumerGroupDescription> descriptions = descriptionsMap.values().stream().toList();
                  return getConsumerGroups(ac, descriptions)
                      .map(g -> Streams.zip(g.stream(), descriptions.stream(), GroupWithDescr::new).toList())
                      .map(g -> sortAndPaginate(g, comparator, pageNum, perPage, sortOrderDto).toList())
                      .map(internalGroups -> internalGroups.stream().map(GroupWithDescr::d).toList());
                }
            );
      }
```

Reply:
Done